### ISSUE TITLE
Auditing

### ISSUE THREAD

**Issue by bergice** [kinglozzer/silverstripe-session-manager#20]
# Overview

We should log more information to retain after-the-fact diagnostic abilities, e.g. to trace back potentially malicious behaviour.

# Acceptance Criteria

* Recommend using the module in tandem with silverstripe/auditor (suggested composer requirement?)
    * Send that module additional events when it is present
* Log the following events
  - Purging sessions
  - 'Remember me' logins (potentially that should be captured outside the session manager module)
* Consider what other events could be logged
* Documentation is updated afterwards describing how much data is logged and stored for later audits.

# Notes
- If we find that the auditor module needs to store more data than it currently does out of the box, this can be covered in a follow-up issue

### Blocked by
- https://github.com/silverstripe/silverstripe-session-manager/pull/25

### PRs
- [x] https://github.com/silverstripe/silverstripe-session-manager/pull/34
- [x] https://github.com/silverstripe/silverstripe-auditor/pull/37
